Jenkinsfile

```diff
@@ -113,8 +138,10 @@ try {
     }
 
     parallel(tasks)
-
     common.setResultIfNotSet(Constants.JSUCCESS)
+    if (releaseBuild == 'Yes') {
+	uploadPackagePyPI()
```

Comment:
Formatting

Reply:
We will not publish in all branches if the branch version of the build changes, then only that will be uploaded as per our naming tags

So for the Master build with Master credentials onetime upload. For the Test credentials also, with versions, only one time upload. 

Can I change to use only Master?